pom.xml

```diff
@@ -380,71 +378,13 @@
     </testResources>
   </build>
 
-  <profiles>
-    <profile>
-      <id>java16</id>
-      <activation>
-        <jdk>1.6</jdk>
-      </activation>
-      <properties>
-        <maven.compiler.testTarget>1.6</maven.compiler.testTarget>
-        <maven.compiler.testSource>1.6</maven.compiler.testSource>
-        <maven.compiler.testCompilerArgument />
-      </properties>
-      <build>
-        <pluginManagement>
-          <plugins>
-            <plugin>
-              <groupId>org.apache.maven.plugins</groupId>
-              <artifactId>maven-compiler-plugin</artifactId>
-              <configuration>
-                <testExcludes>
-                  <testExclude>**/usesjava7/**/*.java</testExclude>
-                  <testExclude>**/usesjava8/**/*.java</testExclude>
-                </testExcludes>
-              </configuration>
-            </plugin>
-            <plugin>
-              <groupId>org.apache.felix</groupId>
-              <artifactId>maven-bundle-plugin</artifactId>
-              <version>2.5.4</version> <!-- For java 6, remain at older version -->
-            </plugin>
-          </plugins>
-        </pluginManagement>
-        <plugins>
-            <plugin>
-              <groupId>org.apache.felix</groupId>
-              <artifactId>maven-bundle-plugin</artifactId>
-              <version>2.5.4</version> <!-- For java 6, remain at older version -->
-            </plugin>
-        </plugins>
-      </build>
-    </profile>
-    <profile>
-      <id>java17</id>
-      <activation>
-        <jdk>1.7</jdk>
-      </activation>
-      <properties>
-        <maven.compiler.testTarget>1.7</maven.compiler.testTarget>
-        <maven.compiler.testSource>1.7</maven.compiler.testSource>
-        <maven.compiler.testCompilerArgument />
-      </properties>
-      <build>
-        <pluginManagement>
-          <plugins>
-            <plugin>
-              <groupId>org.apache.maven.plugins</groupId>
-              <artifactId>maven-compiler-plugin</artifactId>
-              <configuration>
-                <testExcludes>
-                  <testExclude>**/usesjava8/**/*.java</testExclude>
-                </testExcludes>
-              </configuration>
-            </plugin>
-          </plugins>
-        </pluginManagement>
-      </build>
-    </profile>
-  </profiles>
+  <repositories>
```

Comment:
This used to work without problems.  Anyone know what changed to require this?  Do we want to add this to the parent?

Reply:
I've added to refer the mybatis-parent 31-SNAPSHOT on travis-ci. (it is temporary solution until releasing mybatis-parent 31).

>  Do we want to add this to the parent?
No.

Reply:
Thanks.  I find it odd though that using the snapshots used to work fine on travis.  I suspect something changed.  I did fully bring the parent up-to-date just a little while ago.  I've got some questions in it that I need to answer but likely could get it released out soon.